lib/styles/dimensions.dart

```diff
@@ -0,0 +1,13 @@
+abstract class Dimension{
+  static const double blurRadius = 10.0;
+  static const double spreadRadius = 5.0;
+  static const double circleAvatarRadius = 100.0;
+  static const double titleFontSize = 30.0;
+  static const double subtitleFontSize = 15.0;
+  static const double letterSpacing = 2.0;
+  static const double dividerHeight = 2.0;
+  static const double cardSubtitleFontSize = 13.0;
+  static const double sizedBoxHeight = 10.0;
+  static const double sizedBoxWidth = 220.0;
+  static const double horizontalPadding = 60.0;
+}
```

Comment:
Missed an extra line here, do Ctrl + Alt + L to fix it or add it by hand

Reply:
done